docker/docker_tools.py

```diff
@@ -813,8 +814,6 @@ def _build_phase_2_core(
     exec(open(activate_this_path).read(), {"__file__": activate_this_path})
 
     w2p_parent = Path(env.WEB2PY_PATH).parent
-    bookserver_path = get_bookserver_path()
```

Comment:
This does not seem related to the title of this PR, and not getting the bookserver_path seems concerning. What is this about?

Reply:
This variable is never used -- I moved it to `docker_tools_misc.py`. This is cleanup from flake8.

Reply:
(By never used, I mean that it's used only in the next line; `run_bookserver_kwargs` isn't used, making `bookserver_path` also unused).